infokala/models.py

```diff
@@ -1,17 +1,18 @@
 # encoding: utf-8
+from __future__ import unicode_literals
 
 from warnings import warn
 
+from dateutil.tz import tzlocal
```

Comment:
importtien yleinen periaate on ollu "ylhäältä alaspäin kauempaa lähemmäksi", ja ryhmittely on ollu ryhmitellä erillisiin ryhmiinsä

1. standardikirjaston importit
2. frameworkin importit (django)
3. muut 3rd party kirjastojen importit
4. importit saman projektin muista sovelluksista
5. importit samasta sovelluksesta (`from .`)

mulla ny ei oo mitään erityisiä vaatimuksia tän suhteen jatkoa ajatellen kun tätä on kehittäny enimmäkseen @shadikka mut tää import-järjestys on selkeesti muutos aikaisempaan